### PR TITLE
1613 cloud openstack managed cloud

### LINKED ISSUE #1613: /cloud/openstack/managed-cloud/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update /cloud/managed-cloud to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/managed-cloud)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-managed-cloud.png)

## Issue / Card

Fixes #1613

### CODE REVIEW

**Comment by yaili** [comment]
Design review:
- [ ] Can we remove the get in touch block from contextual footer?
- [ ] Contextual footer should be neutral
- [ ] Add more space below the title "BootStack can be customised to your requirements" and "Get a BootStack managed cloud in three easy steps"
- [ ] Can the button in "Get a BootStack managed cloud in three easy steps" be neutral?

**Comment by pmahnke** [comment]
@yaili for 'Can the button in "Get a BootStack managed cloud in three easy steps" be neutral?' I assume you mean higher up in the 'Your cloud, your site, our expertise' section?

I added more space to the other headings as well.

**Comment by yaili** [comment]
@pmahnke I don't see any changes in the demo…

File: templates/cloud/managed-cloud.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 <section class="p-strip--light is-deep is-bordered">
   <div class="row">
-    <div class="col-12">
+    <div class="col-12 u-vertically-spaced--large">
```
Comment (deadlight) [inline templates/cloud/managed-cloud.html]: Vertically-spaced shouldn't be used here. It adds spacing top and bottom when you only need bottom.

Try some of the .u-padding-bottom, maybe .u-padding-bottom--x-large

File: templates/cloud/managed-cloud.html [hunk 2]
```diff
@@ -59,7 +59,7 @@ <h3 class="p-heading--four">Build a hyperconverged cloud with BootStack and Supe
 
 <section class="p-strip--light is-bordered">
   <div class="row">
-    <div class="col-12">
+    <div class="col-12 u-vertically-spaced--large">
```
Comment (deadlight) [inline templates/cloud/managed-cloud.html]: change

File: templates/cloud/managed-cloud.html [hunk 3]
```diff
@@ -124,8 +124,8 @@ <h3 class="p-heading-icon__title">Complete control and no lock-in</h3>
     </div>
   </div>
   <div class="row">
-    <div class="col-12">
-      <p><a href="/cloud/contact-us?product=managed-cloud" class="p-button--brand">Schedule a demo</a></p>
+    <div class="col-12 u-vertically-spaced--large">
```
Comment (deadlight) [inline templates/cloud/managed-cloud.html]: change

File: templates/cloud/managed-cloud.html [hunk 4]
```diff
@@ -159,7 +159,7 @@ <h2 class="p-heading--muted u-align--center">BootStack is trusted by these enter
 
 <section class="p-strip strip-light is-bordered">
   <div class="row">
-    <div class="col-12">
+    <div class="col-12 u-vertically-spaced--large">
```
Comment (deadlight) [inline templates/cloud/managed-cloud.html]: change

File: templates/cloud/managed-cloud.html [hunk 5]
```diff
@@ -215,7 +215,7 @@ <h3 class="p-list-step__title">
 
 <section class="p-strip--light">
   <div class="row">
-    <div class="col-12">
+    <div class="col-12 u-vertically-spaced--large">
```
Comment (deadlight) [inline templates/cloud/managed-cloud.html]: change

**Comment by deadlight** [comment]
@yaili the demo has been updated but I think that u-vertically-spaced shouldn't be used here (or at all, anymore)

**Comment by pmahnke** [comment]
@deadlight replaced u-vertically-spaced--large with the bottom padding

**Pull request merged**